3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,86 @@
+//Age-ify (A future age calculator)
+
+const yearOfBirth = 1995;
+const yearFuture = 2045;
+console.log(typeof yearFuture);
+const curentYear = 2023;
+const age = curentYear - yearOfBirth;
+console.log(age);
+const futureAge = yearFuture - curentYear + age;
+console.log(futureAge);
+console.log(`You will be ${futureAge} years old in ${yearFuture}`);
+
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2018;
+const dogYearFuture = 2045;
+const dogYear = 2023;
+const result = dogYear - dogYearOfBirth;
+console.log(result);
```

Comment:
I suppose you have used this log for debugging. You need to remove console log statements that have been used for debugging from the final version :)